### ISSUE TITLE
Training algorithm for inpainting from scratch

### ISSUE THREAD

**Issue by morteza89** [CompVis/latent-diffusion#197]
Could you please help stepwise on how to train the algorithm on our dataset? Thanks

**Comment by BenjaminIrwin** [comment]
I'm also interested in this. @morteza89 have you had any luck?

**Comment by morteza89** [comment]
> 

Not really. I need to train in on my custom dataset, did all the data preprocessing suggested to do based on the LAMA algorithm but still not lucky to start training...

**Comment by BenjaminIrwin** [comment]
Ok. We're in a very similar position. We will contact you and potentially can collab on this, if you're interested.

**Comment by morteza89** [comment]
> Ok. We're in a very similar position. We will contact you and potentially can collab on this, if you're interested.

Sounds great.

**Comment by DMAgit** [comment]
Any update on this?

**Comment by ustczhouyu** [comment]
Hello, I would like to ask the difference between unconditional LDM and conditional LDM. After the model is trained, is unconditional sampling generate image randomly, but not based on a given image? So, if I want to generate a normal image from a flawed image (without any annotations in the inference phase), should I use conditional LDM? @morteza89 @BenjaminIrwin @asanakoy @pesser

**Comment by ksai2324** [comment]
Any updates on this? I also want to train my own inpainting model, but it seems like there is no documentation available on how to do that
